packages/ui/components/footer/footer.tsx

```diff
@@ -50,7 +50,16 @@ function FooterHeader(props: PropsWithChildren): JSX.Element {
 }
 
 const navigation = {
-  general: [{ name: 'Blog', href: '/' }],
+  general: [
+    {
+      name: 'Blog',
+      href: '/'
+    },
+    {
+      name: 'YouTube',
```

Comment:
Can we add this link to the header as well?
![image](https://user-images.githubusercontent.com/7453394/222131737-dae947e9-c6b5-4f30-af71-6c9795d06a46.png)

Reply:
How about we do that in another pr since its a little bit more difficult than adding a key value pair in `theme.config.js`. We should also fix the lingering slack invites blog pages. I can work on adding them later today.